src/cow-react/modules/limitOrders/containers/LimitOrdersWidget/index.tsx

```diff
@@ -286,8 +286,8 @@ const LimitOrders = React.memo((props: LimitOrdersProps) => {
               )}
 
               <styledEl.FooterBox>
-              <DeadlineInput />
-              {/* {!isWrapOrUnwrap && <styledEl.StyledRateInfo rateInfoParams={rateInfoParams} />} */}
+                <DeadlineInput />
```

Comment:
@fairlighteth you might need to make sure you have the auto-format on in your IDE. I assume, this change came from a previous change you did in another PR which didn't indent the nested elements